tests/tests/da_compression.rs

```diff
@@ -102,14 +104,23 @@ async fn can_fetch_da_compressed_block_from_graphql() {
     let db = &srv.shared.database;
 
     let on_chain_before_execution = db.on_chain().view_at(&0u32.into()).unwrap();
-    let mut tx_inner = db.off_chain().clone().into_transaction();
-    let db_tx = DecompressDbTx {
-        db_tx: DbTx {
-            db_tx: &mut tx_inner,
+    let mut tx_inner = db.compression().clone().into_transaction();
+    let db_tx = DecompressionContext {
+        compression_storage: CompressionStorageWrapper {
+            storage_tx: &mut tx_inner,
         },
         onchain_db: on_chain_before_execution,
     };
-    let decompressed = decompress(compression_config, db_tx, block).await.unwrap();
+    let decompressed = decompress(
+        fuel_core_compression::Config {
+            // todo: better way to do this
```

Comment:
Dangling todo. Is this to be fixed in this PR or should we create a follow-up for it?

Reply:
addressed in 6210a3ba6e

Reply:
Thanks!